content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,27 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/depends-on
+
+Type: Annotation
+
+Example: `config.kubernetes.io/depends-on:`
+
+```yaml
+  annotations:
+    config.kubernetes.io/depends-on: "/namespaces/default/Pod/pod-a,/namespaces/default/Pod/pod-c"
+```
+
+Used on: All objects
+
+This annotation is used on the target object to be modified to configure explicit 
+dependency ordering. The annotation helps to determine the apply order, and dependency 
+reconciliation while applying and deletion and prevents accidental premature deletion.
+
+For example If the annotation `config.kubernetes.io/depends-on: /namespaces/default/Pod/pod-a` is added to `pod-b`,  it identifies that `pod`-a must be successfully applied prior to `pod-b` actuation.
```

Comment:
```suggestion
For example If the annotation `config.kubernetes.io/depends-on: /namespaces/default/Pod/pod-a`
is added to `pod-b`, it identifies that `pod-a` must be successfully applied
prior to `pod-b` actuation.
```

Reply:
We'd better document which tool is attaching this annotation.
A user needs to know when this appears and how to interpret it.